packages/mask/background/services/identity/persona/sign.ts

```diff
@@ -28,6 +29,7 @@ export async function signWithPersona<T>(
                 message: JSON.stringify(message),
                 requestID,
                 identifier: identifier?.toText(),
+                source,
```

Comment:
```suggestion
                origin: location.origin,
```

Reply:
We can either detect the page when the method is being called.

Reply:
It doesn't work, because in the background.

Reply:
Then implement it in the front end.